cfme/tests/automate/test_buttons.py

```diff
@@ -142,7 +124,7 @@ def test_button_on_host(dialog, request):
     button = Button(group=buttongroup,
                     text=fauxfactory.gen_alphanumeric(),
                     hover="btn_hvr_{}".format(fauxfactory.gen_alphanumeric()),
-                    dialog=dialog, system="Request", request="InspectMe")
+                    dialog=dialog.label, system="Request", request="InspectMe")
```

Comment:
Button should already be widgetastic, so with as_fill_value implemented, you should be able to drop the `.label` here as well.